x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/util/lookups.ts

```diff
@@ -9,14 +9,15 @@ import Papa from 'papaparse';
 import type { SiemMigrationResourceData } from '../../../../../../common/siem_migrations/model/common.gen';
 import { initPromisePool } from '../../../../../utils/promise_pool';
 import type { SiemRuleMigrationsClient } from '../../siem_rule_migrations_service';
+import type { SiemDashboardMigrationsClient } from '../../../dashboards/siem_dashboard_migration_service';
 
 interface LookupWithData extends SiemMigrationResourceData {
   data: object[] | null;
 }
 
 export const processLookups = async (
   resources: SiemMigrationResourceData[],
-  ruleMigrationsClient: SiemRuleMigrationsClient
+  migrationsClient: SiemRuleMigrationsClient | SiemDashboardMigrationsClient
```

Comment:
NIT: It looks like we only use the _lookups_ data client inside the function. We could consider passing `lookupsClient: SiemMigrationsDataLookupsClient` parameter instead of the entire migration client.